webapp/publisher/views.py

```diff
@@ -12,12 +12,14 @@
     MacaroonRefreshRequired,
     MissingUsername,
 )
-
+from webapp.decorators import login_required
 
 account = flask.Blueprint(
     "account", __name__, template_folder="/templates", static_folder="/static"
 )
 
+marketo = marketo_api.MarketoApi()
```

Comment:
You seem to be redefining `marketo` in both views, could we just make use of this instance instead?

Reply:
Ah, the two in the methods are from a bad merge, thanks :)!